horovod/mxnet/mpi_ops.cc

```diff
@@ -119,17 +117,18 @@ void DoAllgatherCudaOnCPU(NDArray* tensor, NDArray* output, std::string& name,
   auto enqueue_result = EnqueueTensorAllgather(
       hvd_context, hvd_cpu_tensor, ready_event,
       name, CPU_DEVICE_ID,
-      [hvd_cpu_output, output, cb](const Status& status) {
+      [hvd_cpu_output, output, on_complete](const Status& status) {
         TensorUtil::CopyCPUToCuda(hvd_cpu_output->tensor(), output);
-        cb();
+          on_complete(status.ok() ? nullptr : status.reason().c_str());
```

Comment:
nit: spacing does not align with line above

Reply:
Done